lib/internal/streams/readable.js

```diff
@@ -1015,13 +1020,15 @@ function resume_(stream, state) {
 }
 
 Readable.prototype.pause = function() {
-  debug('call pause flowing=%j', this._readableState.flowing);
-  if (this._readableState.flowing !== false) {
+  const state = this._readableState;
```

Comment:
Are the changes here related to the event stuff or an unrelated refactor?

Reply:
Unrelated. It's in a separate commit.